zuul.d/playbooks/upgrade/run-zed-2023.1.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2024 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Upgrade test
+  hosts: all
+  become: true
+  tasks:
+    - name: Checkout to Zed branch
+      become: true
+      ansible.builtin.shell: |
+        sudo apt-get purge -y snapd
+        git stash
+        git checkout stable/zed
```

Comment:
Can you make this a variable in the upgrade job so we can change the variable and reuse the job?